packages/eslint-plugin/configs/es5.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
 	rules: {
 		'array-bracket-spacing': [ 'error', 'always' ],
+		'array-callback-return': 'error',
```

Comment:
Should we note this rule change in CHANGELOG.md ? It may in-fact be considered a breaking change, which is not hugely problematic since the pending unreleased version of the module is already bumping the major.

Reply:
Yes, please do update the changelog with this change

Reply:
The changelog was added 👍